### ISSUE TITLE
Split transactions aren't handled properly

### ISSUE THREAD

**Issue by peti** [hpdeifel/hledger-iadd#10]
When I choose a template transaction that has multiple postings, `hledger-iadd` will offer only the first two for completion -- unlike `hleder add`.

**Comment by hpdeifel** [comment]
I don't really understand what you mean by "template transaction". Could you please describe the difference in behaviour between `hledger add` and `hledger-iadd` in more detail?

**Comment by peti** [comment]
Just use the journal

~~~
2016/11/24 This is a test
    Foobar                               EUR 1000.00
    Foo                                  EUR -800.00
    Bar                                  EUR -200.00
~~~

and try to add another "This is a test" transaction to it with `hledger-iadd`, then you'll see what I mean.

**Comment by hpdeifel** [comment]
I see what you mean now. `hledger-iadd` *does* suggest all three accounts, just in the wrong order. It also always suggests the balancing amount. Try entering a different amount for the "Bar" account.

This behaviour was partly intended but I think the current `hledger add`-way is indeed better. I'll see what I can do about implementing that.

**Comment by hpdeifel** [comment]
I've changed the account suggestion to be more like `hledger add`. Does this fix your issues with it?

**Comment by peti** [comment]
Does `hledger-iadd-1.1.2` contain those fixes? I don't seem to see any difference in behavior when I tried that version.

**Comment by hpdeifel** [comment]
No, this is currently only available in git. I'd like to test these changes a little more before making a release containing them.

**Comment by hpdeifel** [comment]
I just released `hledger-iadd-1.1.3` containing these fixes
